app/grants/views.py

```diff
@@ -1634,7 +1634,10 @@ def grant_edit(request, grant_id):
         if region:
             grant.region = region
 
+        grant.opt_out_clr = bool(int(request.POST.get('opt_out_clr')))
```

Comment:
have a default as False ?

Reply:
yes in db, but also here `'opt_out_clr': form.opt_out_clr ? '1' : '0'`

string '0' evaluates as python `False` i.e `bool(int(<'0'>))`